<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgrammerMain</class>
 <widget class="QMainWindow" name="ProgrammerMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProgrammerMain</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="groupBoxDeviceTree">
       <property name="title">
        <string>Device Tree: 0 device(s) found</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <property name="spacing">
         <number>3</number>
        </property>
        <item row="0" column="0">
         <widget class="QTreeWidget" name="treeWidget">
          <attribute name="headerHighlightSections">
           <bool>true</bool>
          </attribute>
          <attribute name="headerShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Device</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>File</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Operation</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Status</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Progress</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Log</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <property name="spacing">
         <number>3</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>997</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionUser_Guide"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Rapid_Programmer"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionHarware_settings"/>
    <addaction name="actionDevice"/>
    <addaction name="actionJTAG_Settings"/>
    <addaction name="actionOptions"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDetect"/>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
  </widget>
  <action name="actionDetect">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/detect.png</normaloff>:/images/detect.png</iconset>
   </property>
   <property name="text">
    <string>Detect</string>
   </property>
   <property name="iconText">
    <string>Detect</string>
   </property>
   <property name="toolTip">
    <string>Auto. Detect Device(s)...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout_Rapid_Programmer">
   <property name="text">
    <string>About Rapid Programmer</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/start.png</normaloff>:/images/start.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start programming</string>
   </property>
  </action>
  <action name="actionAdd_File">
   <property name="icon">
    <iconset>
     <normaloff>:/images/icons/microprocessor.png</normaloff>:/images/icons/microprocessor.png</iconset>
   </property>
   <property name="text">
    <string>Add Bitstream File ...</string>
   </property>
  </action>
  <action name="actionProgram">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Program</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
  <action name="actionErase">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Erase</string>
   </property>
  </action>
  <action name="actionVerify">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Verify</string>
   </property>
  </action>
  <action name="actionBlankcheck">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Blankcheck</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionUser_Guide">
   <property name="text">
    <string>User Guide</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/pause.png</normaloff>:/images/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="toolTip">
    <string>Stop programming</string>
   </property>
  </action>
  <action name="actionHarware_settings">
   <property name="text">
    <string>Hardware Settings</string>
   </property>
  </action>
  <action name="actionDevice">
   <property name="text">
    <string>Device</string>
   </property>
  </action>
  <action name="actionJTAG_Settings">
   <property name="text">
    <string>JTAG Settings</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
